crates/miden-objects/src/account/storage/slot/slot_id.rs

```diff
@@ -38,6 +41,16 @@ impl StorageSlotId {
         self.prefix
     }
 
+    /// Computes the [`StorageSlotId`] from a slot name.
+    ///
+    /// The provided `name`'s validity is **not** checked.
+    pub(super) fn compute(name: &str) -> StorageSlotId {
```

Comment:
nit: I'd move this to the "constructors" section and change the return type to `Self`. Also, maybe would rename it to `from_str()` or something like that.